eth-contracts/contracts/DelegateManager.sol

```diff
@@ -360,7 +360,12 @@ contract DelegateManager is InitializableV2 {
             ).div(totalActiveFunds);
 
             // Multiply by deployer cut fraction to calculate reward for SP
-            uint spDeployerCut = (rewardsPriorToSPCut.mul(deployerCut)).div(deployerCutBase);
+            // Operation constructed to perform all multiplication prior to division
+            // uint spDeployerCut = (rewardsPriorToSPCut / deployerCut ) / (deployerCutBase);
```

Comment:
@hareeshnagaraj did you meanto do `rewardsPriorToSPCut * deployerCut`?

Reply:
yes